sdk/communication/azure-communication-chat/pom.xml

```diff
@@ -55,7 +55,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-communication-identity</artifactId>
-      <version>1.1.0-beta1</version> <!-- {x-version-update;com.azure:azure-communication-identity;current} -->
```

Comment:
Why are we removing beta here?

Reply:
I'm just restoring it to what it was before the release hack.